assemblies/assembly-enabling-authentication.adoc

```diff
@@ -1,39 +1,46 @@
-- Additional providers for accessing external resources.
-
-The {product} supports the following authentication providers:
-
-//Auth0:: `auth0`
-//Atlassian:: `atlassian`
-Microsoft Azure::
-`microsoft`
-//Microsoft Azure Easy Auth:: `azure-easyauth`
-//Bitbucket:: `bitbucket`
-//Bitbucket Server:: `bitbucketServer`
-//Cloudflare Access:: `cfaccess`
-GitHub:: `github`
-//GitLab:: `gitlab`
-//Google:: `google`
-//Google IAP:: `gcp-iap`
-Keycloak:: `oidc`
-//Okta:: `okta`
-//OAuth 2 Custom Proxy:: `oauth2Proxy`
-//OneLogin:: `onelogin`
-//SAML:: `saml`
-
-For each provider that you want to use, follow the dedicated procedure to complete the following tasks:
+Your organization certainly requires you to identify and authorize users to give them access to resources, such as {product}.
+
+In {product-short}, authentication and authorization are two separate processes:
+
+. Authentication is defining the user identity, and passing on this information to {product-short}.
+Read following chapters to progressively configure authentication in {product-short}.
+
+. Authorization is defining what actions an authenticated identity can perform in {product-short}.
+See link:{authorization-book-url}[{authorization-book-title}].
+
+To start exploring {product-short} features, do not worry about authentication and authorization: after the default installation, you can authenticate with the Guest user, and use all the application features.
+
+The authentication system in {product-short} is handled by external authentication providers.
+
+{product-short} supports following authentication providers:
+
+* Red Hat Single-Sign On (RHSSO)
+* GitHub
+* Microsoft Azure
+
+To identify users in {product-short}, configure one authentication provider for sign-in and identification:
 
 . Set up the shared secret that the authentication provider and {product} require to communicate.
+
 . Configure {product} to use the authentication provider.
 
+Optionally, to add more information to the user identity, or enable access to additional external resources, configure additional providers for identification.
+
+You can use only one provider for sign-in.
+
+{product-short} is using an internal user catalog to store identity information about the users.
+
+To explore the authentication system, and use {product-short} without authorization policies, you can bypass the user catalog, and start using {product-short} without provisioning the {product-short} user catalog.
+
+To get, store, and update additional information about the users, such as group or team ownership, with the intention to use this information to define authorization policies, provision users and groups in the {product-short} catalog.
+
+{product-short} uses a one way synchronization system to provision user and groups from your authentication system to the {product-short} user catalog.
+Therefore, deleting users and groups by using {product-short} Web UI or REST API might have unintended consequences.
```

Comment:
This section introduces the concept of a user catalog but as a user I may not know anything about what a user catalog is or what it does.

Could we provide a brief explanation or link to a section elsewhere in the product documentation that covers this concept? Also maybe include a paragraph heading to tie this collection of connected sentences together?

Reply:
Defined at line 34.